src/components/flexjar/RutingFlexjar.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Alert,
+  BodyShort,
+  Box,
+  Button,
+  Label,
+  Radio,
+  RadioGroup,
+  Textarea,
+} from '@navikt/ds-react';
+import {
+  FlexjarFeedbackDTO,
+  useFlexjarFeedback,
+} from '@/data/flexjar/useFlexjarFeedback';
+import { ChevronDownIcon, ChevronUpIcon } from '@navikt/aksel-icons';
+import { defaultErrorTexts } from '@/api/errors';
+import { StoreKey, useLocalStorageState } from '@/hooks/useLocalStorageState';
+import * as Amplitude from '@/utils/amplitude';
+import { EventType } from '@/utils/amplitude';
+
+const texts = {
+  apneKnapp: 'Flytter du sykmeldte mellom enheter?',
+  anonym: 'Anonym tilbakemelding',
+  feedbackLabelDescription:
+    'Ikke skriv inn navn eller andre personopplysninger.',
+  followupQuestion: 'Hva er årsaken til/bakgrunnen for flyttingen?',
+  send: 'Send',
+  validation: 'Vennligst velg en tilbakemelding',
+  success: 'Takk for din tilbakemelding!',
+  sporsmal: 'Bruker du Arena til å flytte sykmeldte mellom enheter?',
+};
+
+function logPageView(side: string) {
+  Amplitude.logEvent({
+    type: EventType.PageView,
+    data: { url: window.location.href, sidetittel: side + ' - med Flexjar' },
+  });
+}
+
+enum RadioOption {
+  JA = 'Ja',
+  NEI = 'Nei',
+}
+
+interface FlexjarProps {
+  side: string;
+}
+
+export const RutingFlexjar = ({ side }: FlexjarProps) => {
```

Comment:
Greit å bare følge konvensjoner for nye greier? 😊

```suggestion
interface Props {
  side: string;
}

export default function RutingFlexjar({ side }: FlexjarProps) {
```